### PR TITLE
Add missing css_class to accordion and accordion-group templates

### PR DESCRIPTION
**Pull request by kosdmit**
This PR is based on PR [#169](https://github.com/django-crispy-forms/crispy-bootstrap5/pull/169), and it completely resolves [issue #168 ](https://github.com/django-crispy-forms/crispy-bootstrap5/issues/168).

I've fixed accordion.html and accordion-group.html templates and written a few tests for checking css_classes is rendered correctly.
I've also fix some tests templates, because the first accordion item should have css_class 'active' by default (look [comment](https://github.com/django-crispy-forms/crispy-bootstrap5/issues/168#issuecomment-2200271536) in issue for details.

### CODE REVIEW

**Comment by smithdc1** [comment]
Thanks for the PR!

Just one comment on about this:

>  because the first accordion item should have css_class 'active' by default

Is this the case with Bootstrap 5? I can't see any reference to this class in their [docs](https://getbootstrap.com/docs/5.0/components/accordion/) and trying it locally seems to indicate there are no styles for that class?

**Comment by kosdmit** [comment]
You are right, that is my mistake, bootstrap 5 does not use 'active' class.
But we have probably a legacy method in django-crispy-forms which adds 'active' to css_class string:
```python
class Container(Div):
    def render(self, form, context, template_pack=TEMPLATE_PACK, **kwargs):
        if self.active:
            if "active" not in self.css_class:
                self.css_class += " active"
        else:
            self.css_class = self.css_class.replace("active", "")
        return super().render(form, context, template_pack)
```
I think it's would right to add a separate attr to define activity of item, and do not add this class to css_class string. What do you think?

**Comment by smithdc1** [comment]
I was wondering where this was actually required, seems that it is for [tabs](https://getbootstrap.com/docs/5.0/components/navs-tabs/) rather than accordion. 

Maybe we should move `render()` method you provided above to `Tab` from `Container`? I can't see `active` being on an accordion going back to Bootstrap 3, see [docs](https://getbootstrap.com/docs/3.4/javascript/#collapse-example-accordion) but would need to do some further testing of that.

**Comment by kosdmit** [comment]
Hi @smithdc1, 

I've done some work to close that issue in this template pack:

- Reverted incorrect changes in result's templates. They don't use 'active' class as per the bootstrap docs.
- The tests that were failing b the django-crispy-form reason, get compatible with current lib versions parameter. It also marked as xfail with condition specified by crispy_forms.__version__.

File: tests/test_layout_objects.py
Comment (kosdmit) [inline tests/test_layout_objects.py:1]: Then i was making these tests a little bit more complicated , i was thought we run these tests in couple with several versions of django-crispy-forms :) But actually, it is not.
We should to choose one thing: Test template with different versions of base lib, or make revert updated tests to initial state.
Probably to test templates with different versions of the base lib is not a bad idea, what do you think?

File: tests/test_layout_objects.py [hunk 1]
```diff
@@ -260,8 +262,15 @@ def test_inline_checkboxes_failing(self):
             expected = "inline_checkboxes_failing.html"
         assert parse_form(form) == parse_expected(expected)
 
+    @pytest.mark.parametrize("keep_compatibility", [
+        True,
+        pytest.param(False, marks=pytest.mark.xfail(
+            condition=crispy_forms.__version__ <= "2.2",
+            reason="django-crispy-forms issue #1395",
+        )),
+    ])
```
Comment (kosdmit) [inline tests/test_layout_objects.py:271]: Then i was making these tests a little bit more complicated , i was thought we run these tests in couple with several versions of django-crispy-forms :) But actually, it is not.
We should to choose one thing: Test template with different versions of base lib, or make revert updated tests to initial state.
Probably to test templates with different versions of the base lib is not a bad idea, what do you think?

Comment (smithdc1) [reply tests/test_layout_objects.py:271]: Nice. I was thinking about this. 

I was thinking we should bump the minimum supported version of django-crispy-forms. What do you think?

Comment (kosdmit) [reply tests/test_layout_objects.py]: Good proposal. It's done ✅

**Comment by smithdc1** [comment]
Could you add an entry to the changelog?

File: tox.ini [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist =
-    {py38,py39,py310,py311}-django{42}-crispy{2,-latest},
```
Comment (smithdc1) [inline tox.ini:3]: I'll pull the changes to this file into a separate PR. See #182

**Comment by smithdc1** [comment]
Thank you! 🥇

**Pull request merged**